### PR TITLE
Update default k8s version 1.18.10

### PR DESCRIPTION
**Pull request by jsturtevant**
This updates the default k8s version as well as new crictl version.  I wasn't sure if it should be 1.18.10 or 1.19.3 just let me know which and I will update.

### CODE REVIEW

File: images/capi/packer/config/kubernetes.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "kubernetes_container_registry": "k8s.gcr.io",
   "kubernetes_load_additional_imgs": "false",
   "kubeadm_template": "etc/kubeadm.yml",
-  "crictl_version": "1.16.1",
+  "crictl_version": "1.19.0",
   "crictl_sha256": "19fed421710fccfe58f5573383bb137c19438a9056355556f1a15da8d23b3ad1",
```
Comment (detiber) [inline images/capi/packer/config/kubernetes.json:17]: Does this sha need to be updated as well?

Comment (jsturtevant) [reply images/capi/packer/config/kubernetes.json]: yes. I updated from https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.19.0

**Comment by CecileRobertMichon** [comment]
> I wasn't sure if it should be 1.18.10 or 1.19.3 just let me know which and I will update.

I'm slightly leaning towards n-1 (so 1.18.10 at this point) and I think that's what we had discussed in the past but would like to know what others think. @moshloop / @detiber / @EleanorRigby / @kkeshavamurthy ? (Travis not tagging you on purpose but 
 feel free to chime in if you happen to read this)

**Comment by kkeshavamurthy** [comment]
Leaning towards n-1 as well.

**Comment by detiber** [comment]
> > I wasn't sure if it should be 1.18.10 or 1.19.3 just let me know which and I will update.
> 
> I'm slightly leaning towards n-1 (so 1.18.10 at this point) and I think that's what we had discussed in the past but would like to know what others think. @moshloop / @detiber / @EleanorRigby / @kkeshavamurthy ? (Travis not tagging you on purpose but
> feel free to chime in if you happen to read this)

I'm not overly picky on n or n-1 for the default. At some point we probably need to make it easier to build the latest image for n, n-1 and n-2, without as much work as it takes today, especially as we look to automate building/testing of images. That isn't something that needs to be solved in this PR, though.

**Comment by jsturtevant** [comment]
Sounds like there is a slight preference for 1.18.  I will update for that version then

> At some point we probably need to make it easier to build the latest image for n, n-1 and n-2, without as much work as it takes today, especially as we look to automate building/testing of images. 

I believe https://github.com/kubernetes-sigs/image-builder/issues/18 is the tracking issue

File: images/capi/packer/config/kubernetes.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "kubernetes_series": "v1.17",
-  "kubernetes_semver": "v1.17.11",
-  "kubernetes_rpm_version": "1.17.11-0",
-  "kubernetes_deb_version": "1.17.11-00",
+  "kubernetes_series": "v1.18",
+  "kubernetes_semver": "v1.18.10",
```
Comment (CecileRobertMichon) [inline images/capi/packer/config/kubernetes.json:3]: should this be 1.18.12 now?

**Comment by codenrhoden** [comment]
I know there is a beta out for 1.20.0 now, but we've previously agreed in office hours that our general approach would be to set the default K8s version to the latest version of the n-2 release. Since 1.19.x is still the latest, I'd like to keep this at 1.17.x.

Happy to revisit this if it's causing problems, however. And yes, we should document this. 😆

**Comment by jsturtevant** [comment]
@codenrhoden sounds good. Do you think under https://github.com/kubernetes-sigs/image-builder/blob/master/docs/book/src/capi/capi.md#configuration would be a good place to document it?

**Pull request closed without merging**